theano/gpuarray/dnn.py

```diff
@@ -1393,6 +1393,9 @@ def local_abstractconv_cudnn(node):
     inp1 = node.inputs[0]
     inp2 = node.inputs[1]
 
+    if (node.op.filter_dilation != (1, 1)):
```

Comment:
I've got this on my todo list as well. I actually expect the cuDNN gradient to be slower (especially if we keep the gpu_contiguous on its arguments), but haven't tried yet.